cli_reference/get_started_cli.adoc

```diff
@@ -195,6 +222,13 @@ $ tar -xf <file>
 
 // end::installcli[]
 
+ifdef::openshift-enterprise,openshift-origin[]
+Alternatively, if the cluster administrator has enabled it, you can download and
+unpack the CLI from the
+link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[*About*
+page on the web console].
+endif::[]
```

Comment:
I think this should be split up and put into the respective ifdefs like you have it in the Windows and Mac sections.